invenio_app_ils/circulation/utils.py

```
@@ -30,6 +30,14 @@ def circulation_build_item_ref(loan_pid):
     }
 
 
+def circulation_item_patcher(item_pid, **kwargs):
+    """Patch an item's field(s) and commit the changes."""
+    try:
+        Item.get_record_by_pid(item_pid).patch([kwargs]).commit()
+    except PersistentIdentifierError as ex:
+        current_app.logger.error(ex)
```

This would cause the patcher to fail without raising an error if there was a problem resolving the PID. Would it be better to let the error be raised so the UI can display an error message?